tests/publiccloud/prepare_tools.pm

```diff
@@ -71,6 +71,14 @@ sub run {
     assert_script_run('sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc');
     zypper_call('addrepo --name "Azure CLI" --check https://packages.microsoft.com/yumrepos/azure-cli azure-cli');
     zypper_call('install --from azure-cli -y azure-cli');
+
+    # Install Google Cloud SDK
+    assert_script_run("touch ~/.bashrc");
+    assert_script_run("wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-220.0.0-linux-x86_64.tar.gz");
+    assert_script_run("tar zxf google-cloud-sdk-220.0.0-linux-x86_64.tar.gz google-cloud-sdk");
+    assert_script_run("cd google-cloud-sdk/ && ./install.sh -q");
+    assert_script_run("echo . /root/google-cloud-sdk/completion.bash.inc >> ~/.bashrc");
```

Comment:
this will work even without line 76 , maybe better drop line 76 ?

Reply:
right, ok